sycl/test/regression/vec_value_type.cpp

```diff
@@ -0,0 +1,11 @@
+// RUN: %clangxx -fsycl %s
```

Comment:
```suggestion
// RUN: %clangxx -fsycl -fsyntax-only %s
```
from https://github.com/intel/llvm/blob/sycl/sycl/doc/developer/ContributeToDPCPP.md#dpc-headers-and-runtime-tests
> if you only need to check that compilation succeeds, please use [-fsyntax-only](https://clang.llvm.org/docs/ClangCommandLineReference.html#cmdoption-clang-fsyntax-only) compiler flag for such tests: it instructs the compiler to launch reduced set of commands and produce no output (no need to add -o %t.out).